ISSUE_TEMPLATE.md

```diff
@@ -2,16 +2,17 @@
 *Please read the contribution guidelines first!*
```

Comment:
The guidelines are already linked above the text area. I'm not sure if adding markdown links would benefit the readability of the template header. A quick note regarding known issues is reasonable though...